app/views/enroll_course/index.html.haml

```diff
@@ -0,0 +1,26 @@
+.container
```

Comment:
Is this copied from another location? If so, it should probably be moved instead of duplicated, or if it's used in more than one place, the original file can be explicitly rendered instead of the default rails rendering by path.

Reply:
@ragesoss  Yeah,they were copied from **users** view,I'll definetly modify them and remove the duplicate one